src/emulator/storage/rules/utils.ts

```diff
@@ -65,7 +75,7 @@ export async function isPermitted(opts: {
   }
 
   // Skip auth for UI
-  if (["Bearer owner", "Firebase owner"].includes(opts.authorization || "")) {
+  if (isValidAdminCredentials(opts.authorization || "")) {
```

Comment:
Either check here or on line 99. I prefer here so that `authorization` doesn't have to be optional.

Reply:
done